outputs.tf

```diff
@@ -1,48 +1,48 @@
 // aws_rds_cluster
 output "rds_cluster_arn" {
   description = "The ID of the aurora cluster"
```

Comment:
Since we're now outputting more than one Cluster ARN, the description should change from singular `ID` to `IDs`?

Similar comment to all the other outputs too.

Reply:
We are only outputting one ID. Only one rds cluster is being created